arborcat.module

```diff
@@ -347,6 +347,7 @@ function arborcat_create_pickup_request_record($pickup_request_type, $custom_id,
 
       return $id;
     } catch (Exception $e) {
+      drupal_set_message('An error occurred saving this pickup request. Please try again');
```

Comment:
You can add a second argument of 'error' to drupal_set_message() to make the message display with error styling. (and I know drupal_set_message() will be deprecated in Drupal 9)